### PR TITLE
docs(readme): revamp

### PR DESCRIPTION
**Pull request by wizard-28**
## Approach

Revamps the README.

## Checklist

- [x] I confirm that I have read the [contributing guidelines](https://github.com/pacstall/pacstall/blob/develop/CONTRIBUTING.md), and this pull request is abiding by all the clauses stated in the guideline.

### CODE REVIEW

**Review by saenai255 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,122 +1,129 @@
...
+## Installation
 
-You can run the command below to install Pacstall.
-You can also grab the deb file [here](https://github.com/pacstall/pacstall/releases/latest) but it may be a bit older.
-```bash
-sudo bash -c "$(curl -fsSL https://git.io/JsADh || wget -q https://git.io/JsADh -O -)"
+To install the latest release run:
+
+```console
+$ pip install pacstall
```
Comment (D-Brox) [inline README.md:54]: Shouldn't it be a system package?

Comment (wizard-28) [reply README.md:54]: What do you mean? You want pacstall to be installed with `sudo`?

Comment (D-Brox) [reply README.md:54]: Otherwise `sudo pacstall` wouldn't work, I think

Comment (D-Brox) [reply README.md:54]: It can't find local commands
image

Comment (wizard-28) [reply README.md:54]: So what do you want it to be? Suggest the changes.

Comment (D-Brox) [reply README.md:54]: We would probably have to `sudo pip3 install` or install as a deb package

**Review by wizard-28 — COMMENTED** [review]
@Henryws approved in DMs.

**Pull request merged**